src/battle/overlay_12_0224E4FC.c

```diff
@@ -5541,3 +5542,939 @@ BOOL TryFling(BattleSystem *bsys, BATTLECONTEXT *ctx, int battlerId) {
+                ctx->moveNoMetronome[ctx->battlerIdAttacker] = ctx->moveNoTemp;
+            }
+        }
+    } else {
+        ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns = 0;
+    }
+}
+
+void ov12_02256694(BattleSystem *bsys, BATTLECONTEXT *ctx) {
+    if (GetBattlerHeldItemEffect(ctx, ctx->battlerIdAttacker) == HOLD_EFFECT_BOOST_REPEATED) {
+        if ((ctx->moveStatusFlag & MOVE_STATUS_FAIL) &&
+            ctx->moveNoMetronome[ctx->battlerIdAttacker] == ctx->moveNoTemp &&
+            ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns &&
+            !(ctx->selfTurnData[ctx->battlerIdAttacker].rolloutCount) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_RAGE) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_UPROAR) &&
+            !(ctx->linkStatus & (1 << 9)) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_LOCKED_INTO_MOVE)) {
+            ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns--;
+        }
+    } else {
+        ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns = 0;
+    }
+}
+
+//Related to send out Pokemon crys..?
+int ov12_02256748(BATTLECONTEXT *ctx, int battlerId, int battlerType, BOOL encounter) {
+    int ret;
+    int color;
+    BOOL half;
+    
+    if (encounter == TRUE && (battlerType == 2 || battlerType == 3)) {
+        half = TRUE;
+    } else {
+        half = FALSE;
+    }
+
+    ret = 0;
+    
+    if (half == TRUE) {
+        ret = 0;
+    }
+    
+    color = sub_020880B0(ctx->battleMons[battlerId].hp, ctx->battleMons[battlerId].maxHp, 48);
+    
+    if ((ctx->battleMons[battlerId].status & STATUS_ALL) || (color != 4 && color != 3)) {
+        ret = 11;
+    }
+
+    return ret;
+}
+
+BOOL Battler_CanSelectAction(BATTLECONTEXT *ctx, int battlerId) {
+    BOOL ret = TRUE;
+    
+    if ((ctx->battleMons[battlerId].status2 & STATUS2_RECHARGE) ||
+        (ctx->battleMons[battlerId].status2 & STATUS2_RAGE) ||
+        (ctx->battleMons[battlerId].status2 & STATUS2_UPROAR) ||
+        (ctx->battleMons[battlerId].status2 & STATUS2_LOCKED_INTO_MOVE)) {
+        ret = FALSE;
+    }
+    
+    return ret;
+}
+
+void ov12_022567D4(BattleSystem *bsys, BATTLECONTEXT *ctx, Pokemon *mon) {
+    PlayerProfile *profile = BattleSystem_GetPlayerProfile(bsys, BATTLER_PLAYER);
+    int a3 = ov12_0223AB60(bsys);
+    int terrain = BattleSystem_GetTerrainId(bsys);
+    int ballId;
+    
+    if (BattleSystem_GetBattleType(bsys) & BATTLE_TYPE_9) {
+        ballId = BallToItemId(BattleSystem_GetMonBall(bsys, mon));
+    } else {
+        ballId = ctx->itemTemp;
+    }
+    
+    sub_020720FC(mon, profile, ballId, a3, terrain, HEAP_ID_BATTLE);
+}
+
+u8 ov12_0225682C(BATTLECONTEXT *ctx, int battlerId) {
+    return ctx->unk_2300[battlerId][0];
+}
+
+BOOL BattlerCheckSubstitute(BATTLECONTEXT *ctx, int battlerId) {
+    BOOL ret = FALSE;
+    
+    if (ctx->selfTurnData[battlerId].unk14 & (1 << 3)) {
```

Comment:
can this be a constant?

Reply:
``UNK_CONST_3``

Gonna hold off till I know wth unk14 is lol